build/conf/log4j-server.properties

```diff
@@ -1,6 +1,20 @@
+# Copyright 2019 JanusGraph-Docker Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 log4j.appender.console=org.apache.log4j.ConsoleAppender
 log4j.appender.console.Threshold=INFO
 log4j.appender.console.layout=org.apache.log4j.PatternLayout
 log4j.appender.console.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
 
-log4j.rootLogger=INFO, console
\ No newline at end of file
+log4j.rootLogger=INFO, console
```

Comment:
(nitpick) Do we want to have trailing lines or not?

Reply:
I have no strong opinions on the matter, and happy to go with what you prefer. That said, if you have a preference, let's encode it in `.editorconfig` or similar file so that this is handled automatically for everyone, rather than relying on everyone's local custom config.

Of course, everyone will need to install EditorConfig support in their editors, but at least this will enforce consistency in this repo automatically.

Reply:
I like the idea of adding an `editorconfig`. Do you like to create a ticket?

Reply:
In the interest of keeping this change focused on a single aspect, I've reverted the copyright line change back to simply "JanusGraph" and will submit as-is.

If you have a preference on the newlines, please create a new issue and assign it to me and I'll add an `.editorconfig` config accordingly in a separate PR.

Reply:
Sorry, I wrote my response before seeing yours — GitHub doesn't automatically notify you while you're editing a response if a new comment comes in, just saw it come in via email. I don't know whether you want newlines to be added, or whether you want newlines to be automatically removed, so at best, my ticket would be a placeholder to be filled in.

Please file an issue with any editor-style preferences and I'll do my best to address them.